Firmware/Marlin_main.cpp

```diff
@@ -3430,8 +3430,62 @@ void process_commands()
 	  if (starpos != NULL)
 		  *(starpos) = '\0';
 	  lcd_setstatus(strchr_pointer + 5);
+      custom_message_type = CUSTOM_MSG_TYPE_MSGUPD;
   }
 
+  else if (code_seen("M0 ") || code_seen("M1 "))
```

Comment:
This would be much easier to review if there was a separate commit moving the original piece of code in this place, keeping it intact, even with syntax errors (`case` instead of `else if`). @vojtech-pavlik could you do that please?

Reply:
I split it out and provided better commit messages. I hope that's what you were looking for. I wasn't keen of causing intermediate syntax errors, though, I prefer to keep the code bisectable.

Reply:
Great, thanks!